src/isar/eventhandlers/eventhandler.py

```diff
@@ -38,6 +38,8 @@ def __init__(
         state_name: str,
         event_handler_mappings: List[EventHandlerMapping],
         timers: List[TimeoutHandlerMapping] = [],
+        on_entry: Optional[Callable[[], None]] = None,
+        on_transition: Optional[Callable[[], None]] = None,
```

Comment:
These might be removed/changed in a future refactor, once we have extended/replaced the transitions library. They are useful now to ensure that certain conditions have been met when entering a state, but this can be replaced by more systematic testing.